subsystem-drive/source/main.cpp

```diff
@@ -1,13 +1,32 @@
-#include "../implementations/steer-modes.hpp";
+#include "utility/log.hpp"
+
+#include "../implementations/steer-modes.hpp"
 
 using namespace sjsu::drive;
 
+drive_commands SerialEnterCommands()
+{
+    drive_commands commands;
+    sjsu::LogInfo("Enter in commands");
+    scanf("speed: %d", &commands.speed);
```

Comment:
I'm assuming this is so we can edit what the rover is doing at run time?

Reply:
Yes